src/ansys/aedt/core/maxwell.py

```diff
@@ -953,16 +1017,17 @@ def assign_rotate_motion(
 
     @pyaedt_function_handler(face_list="assignment")
     def assign_voltage(self, assignment, amplitude=1, name=None):
-        """Assign a voltage source to a list of faces in Maxwell 3D or a list of objects or edges in Maxwell 2D.
+        """Assign a voltage excitation to a list of faces in Maxwell 3D or a list of objects or edges in Maxwell 2D.
 
         Parameters
         ----------
         assignment : list
-            List of faces, objects or edges to assign a voltage source to.
+            List of faces, objects or edges to assign a voltage excitation to.
         amplitude : float, optional
             Voltage amplitude in mV. The default is ``1``.
         name : str, optional
-            Name of the boundary. The default is ``None``.
+            Name of the excitation.
+            If not provided, a random name with prefix "Voltage" will be generated.
```

Comment:
```suggestion
            Name of the excitation.
            If not provided, a random name with prefix ``Voltage`` will be generated.
```
@anur7 please use these quotes (````) when mentioning names.